src/plugins/intel_cpu/tests/unit/snippets_transformations/arm/snipptes_mark_skipped.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <gtest/gtest.h>
+#include <subgraph_customizable.hpp>
+#include <snippets_helpers.hpp>
+#include <transformations/snippets/aarch64/pass/snippets_mark_skipped.hpp>
+#include "snippets/pass/tokenization.hpp"
+#include "snippets/pass/collapse_subgraph.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+class SnippetsMarkSkippedTests : public TransformationTestsF {
+public:
+    void run() {
+        ASSERT_TRUE(model);
+        ov::snippets::pass::SnippetsTokenization::Config config = { 1, std::numeric_limits<size_t>::max(), true, true, { 3, 4 } };
+        manager.register_pass<ov::intel_cpu::SnippetsMarkSkipped>();
+        manager.register_pass<ov::snippets::pass::EnumerateNodes>();
+        manager.register_pass<ov::snippets::pass::TokenizeSnippets>(config);
+        //
+        // todo: This is a temporary work-around. remove when MatMul tokenization is supported through general pipeline
+        manager.get_pass_config()->set_callback<ov::snippets::pass::TokenizeSnippets>(
+                [](const std::shared_ptr<const ov::Node>& n) -> bool {
+                        return ov::is_type<const ov::op::v0::MatMul>(n);
+                });
```

Comment:
I think we don't need to have this since arm doesn't support matmul tokenization for now

Reply:
Thanks Alexandra for the comment! I retained this because otherwise the two new added tests smoke_Snippets_SkipMatMulFused_MatMulBiasTwoRelu and SnippetsMarkSkippedTests.smoke_Snippets_SkipMatMulFused_MatMulBiasReluDiv would return error of "Subgraph/SnippetsOpset != MatMul/opset1". Further discussions are always welcomed.

Reply:
Thanks for the explanation! For sure, `TokenizeSnippets` is common pass that tokenizes all supported nodes by Snippets. So we have to add this callback to avoid `MatMul` tokenization.

Reply:
Great! Thanks Alexandra!